### ISSUE TITLE
a NewFromImage function

### ISSUE THREAD

**Issue by diamondburned** [eliukblau/pixterm#11]
It seems like there is no exported `NewFromImage` method, which is really ironic, considering how the unexported function takes in one. I could do a PR if you want.

**Comment by eliukblau** [comment]
Can you elaborate this more? Thank you.

**Comment by diamondburned** [comment]
It's basically adding this:
```go
// NewFromImage creates a new ANSImage from an image.Image.
// Background color is used to fill when image has transparency or dithering mode is enabled.
// Dithering mode is used to specify the way that ANSImage render ANSI-pixels (char/block elements).
func NewFromImage(img image.Image, bg color.Color, dm DitheringMode) (*ANSImage, error) {
	return createANSImage(img, bg, dm)
}
```

**Comment by eliukblau** [comment]
Sounds good. Probably I completely forgot this function signature at the time. If you wish, you can do the PR and I will include it. It will be included for the upcomming version, by the way. I've been a little busy, but I'll try to release it soon.

**Comment by diamondburned** [comment]
Can you make a branch with those 2 PRs so I could PR on them?

**Comment by eliukblau** [comment]
Put your PR based on master. I will manipulate branchs later, etc. Don't worry about that ;)

Only one PR with this topic, of course. 👍

**Comment by diamondburned** [comment]
`master` doesn't compile, as far as I've checked

**Comment by diamondburned** [comment]
Here you go anyway: https://github.com/eliukblau/pixterm/pull/13

**Comment by eliukblau** [comment]
Master compiles perfectly fine, if you don't update dependencies. All dependencies are included in vendor directory, managed in this case by Glide dependency manager.

**Comment by eliukblau** [comment]
Ok, merged #13 😀. Thanks for contributing. This will be appear in upcoming v1.3.0 👍

**Comment by eliukblau** [comment]
Solved in [v1.3.0](https://github.com/eliukblau/pixterm/releases/tag/v1.3.0)
@diamondburned check you name in `-credits` option 😉
